database/migrations/2024_12_24_204941_add_avatar_url_and_description_columns_to_streamers_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('streamers', function (Blueprint $table) {
+            $table->string('avatar_url')->nullable();
```

Comment:
No primary key?
and avatar_url is nullable, so you could get a bunch of null records in your database